plugins/flytekit-dask/flytekitplugins/dask/models.py

```diff
@@ -0,0 +1,129 @@
+from typing import Optional
+
+from flyteidl.plugins import dask_pb2 as _dask_task
+
+from flytekit.models import common as _common
+from flytekit.models import task as _task
+
+
+class Scheduler(_common.FlyteIdlEntity):
+    """
+    Configuration for the scheduler pod
+
+    :param image: Optional image to use.
+    :param resources: Optional resources to use.
+    """
+
+    def __init__(self, image: Optional[str], resources: Optional[_task.Resources]):
+        self._image = image
+        self._resources = resources
+
+    @property
+    def image(self) -> Optional[str]:
+        """
+        :return: The optional image for the scheduler pod
+        """
+        return self._image
+
+    @property
+    def resources(self) -> Optional[_task.Resources]:
+        """
+        :return: Optional resources for the scheduler pod
+        """
+        return self._resources
+
+    def to_flyte_idl(self) -> _dask_task.DaskScheduler:
+        """
+        :return: The scheduler spec serialized to protobuf
+        """
+        return _dask_task.DaskScheduler(
+            image=self.image,
+            resources=self.resources.to_flyte_idl() if self.resources else None,
+        )
+
+
+class WorkerGroup(_common.FlyteIdlEntity):
+    """
+    Configuration for a dask worker group
+
+    :param number_of_workers:Number of workers in the group
+    :param image: Optional image to use for the pods of the worker group
+    :param resources: Optional resources to use for the pods of the worker group
+    """
+
+    def __init__(self, number_of_workers: int, image: Optional[str], resources: Optional[_task.Resources]):
```

Comment:
```suggestion
    def __init__(self, number_of_workers: int, image: Optional[str]=None, resources: Optional[_task.Resources]=None):
```

Reply:
Fixed in dab1bc635f91041e849e0bdfc03e187846a41de7